common/src/main/java/com/microsoft/identity/common/internal/commands/BaseCommand.java

```diff
@@ -42,7 +42,6 @@
     @EqualsAndHashCode.Exclude
     private CommandCallback callback;
 
-    @EqualsAndHashCode.Exclude
     private List<BaseController> controllers;
```

Comment:
How many controllers can we have?

Reply:
AFAIK, we only have 2 max at any given time (with the current design). i.e one for local flow and the other for broker flow.